xbmc/utils/CryptThreading.cpp

```diff
@@ -31,11 +32,20 @@ void lock_callback(int mode, int type, const char* file, int line)
     getlock(type)->unlock();
 }
 
+#if KODI_OPENSSL_USE_THREADID
+void thread_id(CRYPTO_THREADID *tid)
+{
+  // C-style cast required due to vastly differing native ID return types
+  CRYPTO_THREADID_set_numeric(tid, (unsigned long)CThread::GetCurrentThreadId());
+}
+#else
+
```

Comment:
```suggestion
```

Reply:
Will do the changes at once when there is a decision in the indentation question.